tatus

```diff
@@ -0,0 +1,136 @@
+[33m249107ec[m[33m ([m[1;36mHEAD -> [m[1;32mUPDATE_MODULE_FILES[m[33m)[m Not needed config file.
```

Comment:
What's this file?

Reply:
resources/stubs/modules/route-provider.stub

Reply:
The not needed config file was my mistake. I just wanted to remove the addon module added in the .json but instead I deleted the file. Sorry.
By the way, I have reuploaded the file from dev master